### PR TITLE
Add screenshot url to the manifest

### PR DESCRIPTION
**Pull request by EvanHerman**
@fjarrett Please review.

I've gone ahead and updated the manifest to include the URL to the screenshots for use in the theme tab.

### CODE REVIEW

**Comment by frankiejarrett** [comment]
@EvanHerman Did you already test to see if these changes break the [`Theme_Updates`](https://github.secureserver.net/WPPaaS/wp-paas-system-plugin/blob/8aeb64a29ab3787dd548a8b60df85933da242e37/gd-system-plugin/includes/class-theme-updates.php) class in the System Plugin?

**Comment by EvanHerman** [comment]
@fjarrett I just tested and it works just fine when not on the VPN. Otherwise it times out.

File: manifest.json [hunk 1]
```diff
@@ -3,60 +3,70 @@
     "theme": "activation",
     "new_version": "1.1.1",
     "url": "http://wpnux.godaddy.com/?theme=activation",
-    "package": "https://github.com/godaddy/wp-activation-theme/releases/download/v1.1.1/activation.zip"
+    "package": "https://github.com/godaddy/wp-activation-theme/releases/download/v1.1.1/activation.zip",
+    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-activation-theme/master/screenshot.png"
```
Comment (frankiejarrett) [inline manifest.json]: @EvanHerman The `screenshot` should reference the release tag, not `master`.

e.g. https://raw.githubusercontent.com/godaddy/wp-activation-theme/v1.1.1/screenshot.png

**Review by frankiejarrett — CHANGES_REQUESTED** [review]
@EvanHerman Please add a unit test for `testScreenshotsReachable()` and an `assertStringEndsWith( 'screenshot.png' )` in the `testThemeData()` method.

**Review by frankiejarrett — APPROVED** [review]

**Pull request merged**